source/administration/config-settings.rst

```diff
@@ -1078,6 +1078,41 @@ This filter uses the permissions of the **Bind Username** account to execute the
 | This feature's ``config.json`` setting is ``"UserFilter": ""`` with string input.                                                                                    |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Group Filter
```

Comment:
Just a reminder that changing a filter can cause groups to be deleted. The group would be undeleted upon the filter being changed back but in the future another effect could be that users may be removed from teams and channels - and possibly make the channels and teams unaccessible, at least temporarily. 

I just wanted to mention it @wiersgallak, in case you decide that it's worth mentioning.